modules/op-resolver-community-tasks-ref.adoc

```diff
@@ -0,0 +1,411 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-community-tasks-ref_{context}"]
+= Community tasks provided in the {pipelines-shortname} namespace
+
+By default, an {pipelines-shortname} installation includes a set of community tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace.
+
+[discrete]
+[id="op-ctaskref-argocd-task-sync-and-wait_{context}"]
+== argocd-task-sync-and-wait
+The `argocd-task-sync-and-wait` community task deploys an Argo CD application and waits for it to be healthy. To do so, it requires the address of the Argo CD server and authentication information, either a username and password or an authentication token.
+
+.Example usage of the `argocd-task-sync-and-wait` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: argocd-task-sync-and-wait
+spec:
+  tasks:
+    - name: argocd-task-sync-and-wait
+      params:
+        - name: application-name
+          value: example_app_name
+        - name: revision
+          value: HEAD
+        - name: flags
+          value: '--'
+        - name: argocd-version
+          value: v2.2.2
+      taskRef:
+        kind: Task
+        name: argocd-task-sync-and-wait
+----
+
+.Supported parameters for the `argocd-task-sync-and-wait` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`application-name` |The name of the application to deploy.|
```

Comment:
Remove "The" here and in the other parameters, I think we don't normally use an article at the start of a parameter description in the table (compare to how the non-community tasks are done)